xknx/core/telegram_queue.py

```diff
@@ -36,20 +36,25 @@ def __init__(
             self,
             callback: "AsyncTelegramCallback",
             address_filters: Optional[List[AddressFilter]] = None,
+            group_addresses: Optional[List[GroupAddress]] = None,
         ):
             """Initialize Callback class."""
             self.callback = callback
-            self.address_filters = address_filters
+            self._match_all = address_filters is None and group_addresses is None
```

Comment:
This is kind of hacky 😬
I guess if Callback is instantiated with no filters (not even empty Lists) its not meant to be changed. 
I'm open for suggestions 😉